test/cpp/lazy/test_misc.cpp

```diff
@@ -0,0 +1,25 @@
+#include <gtest/gtest.h>
+#include <string>
+
+#include "torch/csrc/lazy/core/hash.h"
+#include "torch/csrc/lazy/core/int128.h"
+
+namespace torch {
+namespace lazy {
+
+TEST(HashTest, Sanity) {
+    // repeatable
+    auto str_a = std::string("Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut at suscipit purus.");
```

Comment:
Ideally we also want to test types like `c10::ScalarType` where we explicitly support, but I realized pt/xla never had those.... so not a hard requirement 😄

Reply:
Yea, good point- i'll add coverage for a lot of types.